src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -16,22 +16,30 @@ import './WebexRemoteMedia.scss';
  * NOTE: waiting for the UX for a design on what to display if there is no remote video
  */
 export default function WebexRemoteMedia({meetingID}) {
-  const {remoteAudio, remoteVideo} = useMeeting(meetingID);
+  const {remoteAudio, remoteVideo, error} = useMeeting(meetingID);
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const hasMedia = !!(remoteAudio || remoteVideo);
 
   return (
-    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`}>
-      {!hasMedia ? (
-        <Badge rounded>
-          <Spinner size={16} />
-          <div>Connecting</div>
-        </Badge>
-      ) : null}
-      {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
-      {remoteAudio ? <audio ref={audioRef} autoPlay /> : null}
-    </div>
+    <>
```

Comment:
Can you update this to be an explicit fragment? I think that's how it's done everywhere else
```suggestion
    <React.Fragment>
```

Reply:
Sure

Reply:
Updated!